### ISSUE TITLE
I can't upload or edit files in FTP Server

### ISSUE THREAD

**Issue by rcrIbercomp** [Wiznet/RP2040-HAT-C#20]
Hi, 
I am testing the W5x00_ftp_server.c in a W5100S-EVB-Pico, and I can't upload files to the FTP Server or edit the ones that exists there.

![directorios](https://github.com/Wiznet/RP2040-HAT-C/assets/137757141/32a3d8bd-786c-48a7-a09f-d5e3ec4a7437)

I can build the example, burn it on the RPI and connect through FTP with WinSCP. I can also download the test.txt file in my computer.

This is the trace I get:
--> When Download file:
![download](https://github.com/Wiznet/RP2040-HAT-C/assets/137757141/477de27b-41e9-45c3-a841-0a727cd16bd8)

--> When upload file:
![upload](https://github.com/Wiznet/RP2040-HAT-C/assets/137757141/e6b62059-d154-46ec-a43f-e0a62d9fc63e)

--> When creating file in the FTP Server:
![newFile](https://github.com/Wiznet/RP2040-HAT-C/assets/137757141/c8b419ff-01e4-491a-bc06-8bfb1d1c0672)

Also when trying to edit the new file in the FTP Server, I get connection error.

Any idea of what is happening?

Kind Regards

**Comment by keitarusm** [comment]
I hope you already figured this out, but the FTP example provided doesn't do anything with files, it's only the comms interface. The actual code for the FTP server in ftpd.c, which is part of the ioLibrary_Driver repo and not the RP2040-HAT-C repo, shows this on or about line 295 where you'll see that the response given to the FTP client is hardcoded in. 
Similarly, when you try and store a file the server code will dutifully listen to the transmission, storing it in a buffer temporarily, responding to the client that the file was successfully transferred(regardless of conditions). It will respond with the same buffer it used to 'listen' with, now having overwritten the first bytes with the response string. 

tldr; There is no file handling in the example, it is left as an exercise for the reader.
